src/components/Alert/Alert.tsx

```
@@ -0,0 +1,75 @@
+} as const;
+
+const TEXT_COLOR_CLASSES = {
+    default: 'text-950',
+    info: 'text-info-800',
+    success: 'text-success-800',
+    warning: 'text-warning-800',
+    error: 'text-error-800',
+} as const;
+
+export const Alert = ({
+    variant = 'solid',
```

ESLint

[error] Delete `··`

(prettier/prettier)